wallet/src/de/schildbach/wallet/payments/SendCoinsTaskRunner.kt

```diff
@@ -81,6 +88,10 @@ class SendCoinsTaskRunner @Inject constructor(
                 coinJoinSend = mode != CoinJoinMode.NONE
             }
             .launchIn(coroutineScope)
+
+        FirebaseInstallations.getInstance().id.addOnCompleteListener { task ->
+            firebaseInstallationId = if (task.isSuccessful) task.result else ""
+        }
```

Comment:
`firebaseInstallationId` doesn't seem to be used anywhere?

Reply:
You are right, we don't need to specify this manually.